e2e/generate-test.mjs

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-console, import/no-extraneous-dependencies */
```

Comment:
I assume this script needs to be run for every new test to generate all the code/html needed for that test and that this generated code has to be checked into git. It would generate a lot of code that needs to be maintained in the future in case of changes. Could it be instead implemented as a function or module that is run for every test (kind of like a before hook) that will provide each test with the needed files on-the-fly?

Reply:
We no longer need to generate files in order for tests to work like we used to. Now we simply import things and it will resolve to the yarn workspace. This is ran if you want the boilerplate of a new test, something that is ran once, not for every test.

Reply:
Oh so this script needs to be run by the developer and commited to git for every new test the developer wants to write? Then I’m still wondering if there’s an alternative way, because it will lead to a lot ofduplication and boiler plate, if every new test needs this.

or are you saying this script is to be run only once in the lifetime of uppy codebase? If so, i’m not sure what’s the value of keeping such a script

Reply:
I would suggest checking out the branch and running it to see what it does. To me it's valuable. But given that this wasn't immediately clear, I should perhaps update the docs better.

Reply:
I understand it will generate files under `client/` and a sample test file. I think what I'm not totally understanding yet, is how we are supposed to write new tests. If I or a contributor now want to add a test for a untested feature, like say upload a file from dropbox, should I then run generate-test.mjs to generate a new set of files for my test? And another feature, do I run it again? Or should I add an `it` statement into say https://github.com/transloadit/uppy/blob/956cd92eca9470661d948992278841f6d0f56cd9/e2e/cypress/integration/dashboard-transloadit.spec.ts and start adding my logic there? maybe we need to document this